assets/src/edit-story/components/dropzone/utils/getDragType.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Get Drag Type from event.
+ *
+ * @param {Array} dataTransfer Array / List of type drag event.
```

Comment:
Wouldn't it be correct to use something like this?

```
@param {Object} event Drag and drop event.
@param {DataTransfer} event.dataTransfer The event's data transfer object.
```

Reply:
Fix https://github.com/google/web-stories-wp/pull/376/commits/f780d2d59df2333e442622dfadabe58c8bcd71c4